Model/Cron.php

```diff
@@ -1890,7 +1890,7 @@ protected function addRefundFailedNotice(){
         $this->notifierPool->addNotice(
             __("Adyen: Refund for order #%1 has failed", $this->_merchantReference),
             __("Reason: %1 | PSPReference: %2 | You can go to Adyen Customer Area and trigger this refund manually or contact our support.", $this->_reason, $this->_pspReference),
-            sprintf("https://ca-%s.adyen.com/ca/ca/accounts/showTx.shtml?pspReference=%s", $this->_live === 'true' ? 'live' : 'test',  $this->_pspReference)
+            \Adyen\Util\Util::getPspReferenceSearchUrl($this->_pspReference, $this->_live === 'false' ? 'test' : 'live')
```

Comment:
don't see this method in the PR was it already available ?

Reply:
better not to use ? : but if else statement can we move this into this method?

Reply:
Moved the method to adyen/module-payment Data helper. Also, I changed it to receive the variable from the notification and use live/test based on a if else statement